### PR TITLE
Updating organizer info in about page 

### PR DESCRIPTION
**Pull request by fvntr**
Updating copy 
---

## ✅️ By submitting this PR, I have verified the following

* [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
* [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/CONTRIBUTING.md) 🔍️
* [x] Added my name to the bottom of the list under the **Credits** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my website or GitHub profile 👥️

### CODE REVIEW

**Comment by fvntr** [comment]
I think I got everyone. Maybe the list needs to be update, but this is a better reflection of who is organizing at them moment.

File: site/content/sponsors/_index.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ description: Sponsors and partners who help provide an inclusive community.
 
 ## Location and Event Sponsors
 
-The following companies have hosted our events (in the past or present) and have provided us with snacks and drinks for our members.
+The following companies have hosted our events (in the past or present) and have donated resources to our community:
```
Comment (fvcproductions) [inline site/content/sponsors/_index.md]: Should be period instead of colon at end.

File: site/content/about/_index.md [hunk 2]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
 ### Developers
 
 Our developers maintain this website. You can [contribute](https://github.com/techqueria/website) as well!
 
-- [Frances Coronel](https://www.linkedin.com/in/fvcproductions), software engineer at Slack
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
-
+- [Frances Coronel](https://www.linkedin.com/in/fvcproductions), Software engineer at Slack
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & Senior Developer at Slingshot Health
+- [Phillip Ventura](https://www.linkedin.com/in/fvntr/), Software Engineer
```
Comment (fvcproductions) [inline site/content/about/_index.md]: Should be a space before line break.

File: site/content/about/_index.md [hunk 3]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
+- [CJ Joulain](https://www.linkedin.com/in/cjoulain) Technical Support Engineer at npm
 
 ### Developers
 
 Our developers maintain this website. You can [contribute](https://github.com/techqueria/website) as well!
 
-- [Frances Coronel](https://www.linkedin.com/in/fvcproductions), software engineer at Slack
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
-
+- [Frances Coronel](https://www.linkedin.com/in/fvcproductions), Software engineer at Slack
```
Comment (fvcproductions) [inline site/content/about/_index.md]: Should be "Engineer" instead of "engineer" for consistency

File: site/content/about/_index.md [hunk 4]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
+- [Kevin Hernandez](https://www.linkedin.com/in/kevdez) UI Developer at WalmartLabs
+- [Phillip Ventura](https://www.linkedin.com/in/fvntr/), Software Engineer
 
 #### Los Angeles
 
-- [Esteban Uribe](https://www.linkedin.com/in/estebanuribe/), co-founder of Techqueria's LA chapter & iOS Engineer at Tinder
-- [Hector Carmona](https://www.linkedin.com/in/hectoroddincarmona/), software engineer at Google
-- [Josue Bustos](https://www.linkedin.com/in/josuebustos), technical evangelist at Samsung
+- [Esteban Uribe](https://www.linkedin.com/in/estebanuribe/), Co-founder of Techqueria's LA chapter & iOS Engineer at Tinder
```
Comment (fvcproductions) [inline site/content/about/_index.md]: I learned Co-founder is preferred - so looks good!

https://english.stackexchange.com/questions/62117/co-founder-co-founder-or-cofounder

File: site/content/about/_index.md [hunk 5]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
 
 ### Organizers
 
 #### Bay Area
-
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
-- [David Hincapie](https://www.linkedin.com/in/davidohincapie/), product specialist at Oculus VR
-- [Philip Ventura](https://www.linkedin.com/in/fvntr/), software engineer at Lending Club
+- [Rubi Sanchez](https://www.linkedin.com/in/rubisanchez), Product Manager at AdRoll
```
Comment (fvcproductions) [inline site/content/about/_index.md]: There should be a space below any headers so it's consistent with rest of headers.

File: site/content/about/_index.md [hunk 6]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
 
 ## Team
 
-Techqueria is possible thanks to the work of lots of people, organizers, volunteers, mentors, sponsors, and advisors; and we're thankful for all of them.
+Techqueria is an all-volunteer organiztion that is made possible thanks to the work of lots of organizers, mentors, sponsors, and advisors. Thank you all!
```
Comment (fvcproductions) [inline site/content/about/_index.md]: I think the "Thank you all" message isn't needed as we thank them by naming them throughout the rest of the page.

File: site/content/about/_index.md [hunk 7]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
+- [Esteban Uribe](https://www.linkedin.com/in/estebanuribe/), Co-founder of Techqueria's LA chapter & iOS Engineer at Tinder
+- [Hector Carmona](https://www.linkedin.com/in/hectoroddincarmona/), Software Engineer at Google
+- [Josue Bustos](https://www.linkedin.com/in/josuebustos), Technical Evangelist at Samsung
 
 #### New York
 
-- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), associate at Booz Allen Hamilton
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
+- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), Associate at Booz Allen Hamilton
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & senior developer at Slingshot Health
```
Comment (fvcproductions) [inline site/content/about/_index.md]: senior developer should be capitalized like other mention

File: site/content/about/_index.md [hunk 8]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
 #### Bay Area
-
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
-- [David Hincapie](https://www.linkedin.com/in/davidohincapie/), product specialist at Oculus VR
-- [Philip Ventura](https://www.linkedin.com/in/fvntr/), software engineer at Lending Club
+- [Rubi Sanchez](https://www.linkedin.com/in/rubisanchez), Product Manager at AdRoll
+- [Dalia Icedo](https://www.linkedin.com/in/daliaicedo), Freelance Product Designer
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
+- [Kevin Hernandez](https://www.linkedin.com/in/kevdez) UI Developer at WalmartLabs
+- [Phillip Ventura](https://www.linkedin.com/in/fvntr/), Software Engineer
```
Comment (fvcproductions) [inline site/content/about/_index.md]: Should be Sofware Engineer at xyz for Shashi and you Phillip

File: site/content/about/_index.md [hunk 9]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
 
-Techqueria is possible thanks to the work of lots of people, organizers, volunteers, mentors, sponsors, and advisors; and we're thankful for all of them.
+Techqueria is an all-volunteer organiztion that is made possible thanks to the work of lots of organizers, mentors, sponsors, and advisors. Thank you all!
 
 ### Co-Founders
 
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & Senior Developer at Slingshot Health
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
```
Comment (fvcproductions) [inline site/content/about/_index.md]: Should be Software Engineer for xyz so it's consistent

File: site/content/about/_index.md [hunk 10]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
 
-- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), associate at Booz Allen Hamilton
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
+- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), Associate at Booz Allen Hamilton
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & senior developer at Slingshot Health
+
+### Austin
+- [Andrea Flores](https://www.linkedin.com/in/andreaf), PhD Student at The University of Texas at Austin School of Information
+
+### Past Organizers
```
Comment (fvcproductions) [inline site/content/about/_index.md]: Past organizers should be underneath developers header so the developers are current

File: site/content/about/_index.md [hunk 11]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
 
 #### New York
 
-- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), associate at Booz Allen Hamilton
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
+- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), Associate at Booz Allen Hamilton
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & senior developer at Slingshot Health
+
+### Austin
+- [Andrea Flores](https://www.linkedin.com/in/andreaf), PhD Student at The University of Texas at Austin School of Information
```
Comment (fvcproductions) [inline site/content/about/_index.md]: Austin header should have space under

File: site/content/about/_index.md [hunk 12]
```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
...
+
+### Past Organizers
+
+- [Kimberly Muñoz](https://www.linkedin.com/in/kimberlymunoz) Front End Developer at Slack
+- [CJ Joulain](https://www.linkedin.com/in/cjoulain) Technical Support Engineer at npm
```
Comment (fvcproductions) [inline site/content/about/_index.md]: There should be commas before position

**Comment by fvntr** [comment]
will update asap

**Comment by fvcproductions** [comment]
@fvntr any update for this?

**Comment by fvntr** [comment]
@fvcproductions changes implemented

**Comment by fvntr** [comment]
please ignore shashi's place of work since he is currently working on a startup idea

File: site/content/about/_index.md (hunk 9 above)
Comment (fvntr) [reply site/content/about/_index.md]: He isnt currently working for anyone

**Pull request merged**